BoxOffice/View/BoxOfficeCollectionViewCell.swift

```diff
@@ -0,0 +1,176 @@
+        return label
+    }()
+    
+    private let rankVariationLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.preferredFont(forTextStyle: .body)
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+    
+    private let movieNameLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.font = UIFont.preferredFont(forTextStyle: .title2)
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+    
+    private let audienceNumberLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.font = UIFont.preferredFont(forTextStyle: .body)
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+    
+    func configureCell(boxOfficeItem: BoxOfficeItem) {
+        addSubviews()
+        setUpConstraints()
+        configureCellLabels(by: boxOfficeItem)
+    }
+    
+    private func configureCellLabels(by boxOfficeItem: BoxOfficeItem) {
+        rankLabel.text = boxOfficeItem.rank
+        rankVariationLabel.attributedText = getRankVariationText(boxOfficeItem: boxOfficeItem)
+        movieNameLabel.text = boxOfficeItem.movieName
+        audienceNumberLabel.text = "오늘 \(boxOfficeItem.audienceCount.decimalFormat) / 총 \(boxOfficeItem.accumulatedAudienceCount.decimalFormat)"
+    }
+    
+    private func getRankVariationText(boxOfficeItem: BoxOfficeItem) -> NSMutableAttributedString? {
+        let amountOfRankChange = boxOfficeItem.amountOfRankChange
+        let oldAndNew = boxOfficeItem.rankOldAndNew
+        
+        guard let rankChange = RankChangeState(amountOfRankChange, oldAndNew) else {
+            return nil
+        }
+        
+        switch rankChange {
+        case .신작:
+            let text = "신작"
+            let attribute: [NSAttributedString.Key: Any] = [.foregroundColor: UIColor.red]
+            
+            return NSMutableAttributedString(string: text, attributes: attribute)
+        case .변동없음:
+            return NSMutableAttributedString(string: "-")
+        case .상승:
+            let text = "▲" + amountOfRankChange
+            let string = NSMutableAttributedString(string: text)
+            
+            string.addAttributes([.foregroundColor: UIColor.red], range: NSRange(location: 0, length: 1))
+            
+            return string
+        case .하락:
+            let text = "▼" + amountOfRankChange.dropFirst()
+            let string = NSMutableAttributedString(string: text)
+            
+            string.addAttributes([.foregroundColor: UIColor.blue], range: NSRange(location: 0, length: 1))
+            
+            return string
+        }
+    }
+}
+
+// MARK: - Add Subviews
+extension BoxOfficeCollectionViewCell {
+    private func addSubviews() {
+        [rankLabel, rankVariationLabel, movieNameLabel, audienceNumberLabel].forEach {
+            addSubview($0)
+        }
```

Comment:
권장하지는 모르겠지만.. 정말 짧게 쓰려면 이렇게도 돼요 ㅎ
```suggestion
        [rankLabel, rankVariationLabel, movieNameLabel, audienceNumberLabel].forEach(addSubview)
```

Reply:
좋은 방법 알려주셔서 감사합니다.😊
forEach가 매개변수 하나인 클로저를 파라미터로 받기 때문에 가능한 것이었군요!
파라미터 두 개를 받는 클로저를 만들어 실행해보니 실패한 것을 확인했습니다.
사소하지만 중요한 사실을 상기하고 갑니다 😊